### PR TITLE
Redesign

### PR DESCRIPTION
**Pull request by bcardarella**

### CODE REVIEW

File: source/posts/2015-05-20-the-new-dockyard-dot-com.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+a company. We've always been considered an *engineering* company but we
+are now considering *ourselves* a **User Experience** company. All of
+our design and engineering decisions go into delivering a modern user
+experience for web application to our clients.
+
+The new website is also built how we believe modern web applications
+should be built: with [Ember.js](http://emberjs.com) on the front-end
+and [Phoenix](http://phoenixframework.org) on the backend.
+
+Over the next few weeks we're going to share the story on how to rebuilt
```
Comment (rwjblue) [inline source/posts/2015-05-20-the-new-dockyard-dot-com.md]: > the story on how to rebuilt

Doesn't sound quite right.  Should it be `"the story on how we rebuilt"`?

File: source/posts/2015-05-20-the-new-dockyard-dot-com.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+several other libraries, in order to get the experience just right.
+[Amanda Cheung](http://twitter.com/acacheung), our Lead UX Developer, will share what efforts went into the implementation of
+the design, how we build modern response markup. Finally you'll hear
+from [Jon Lacks](http://twitter.com/jon_lacks), our Project Manager, will show how we managed the rebuild
+of our own website using the exact same processes we use on our client
+projects.
+
+Along with these stories we'll not just tell you how we did things, we
+plan to show you. We'll be open sourcing all of the code for the Ember
+front and as well as the Phoenix back end in the coming weeks. There are
```
Comment (poteto) [inline source/posts/2015-05-20-the-new-dockyard-dot-com.md]: front ~~and~~ end

File: source/posts/2015-05-20-the-new-dockyard-dot-com.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+tags: business, announcement
+---
+
+I'm proud to announce the new [DockYard.com](http://dockyard.com)
+
+![](http://i.imgur.com/D3KeY1b.jpg)
+
+With this relaunch of our website we are also changing our direction as
+a company. We've always been considered an *engineering* company but we
+are now considering *ourselves* a **User Experience** company. All of
```
Comment (poteto) [inline source/posts/2015-05-20-the-new-dockyard-dot-com.md]: Not sure if `ourselves` needs to be italicised

File: source/posts/2015-05-20-the-new-dockyard-dot-com.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+our design and engineering decisions go into delivering a modern user
+experience for web application to our clients.
```
Comment (poteto) [inline source/posts/2015-05-20-the-new-dockyard-dot-com.md]: web application`s` ~~to~~ for our clients

File: source/posts/2015-05-20-the-new-dockyard-dot-com.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+The new website is also built how we believe modern web applications
+should be built: with [Ember.js](http://emberjs.com) on the front-end
```
Comment (poteto) [inline source/posts/2015-05-20-the-new-dockyard-dot-com.md]: inconsistent – `front-end` and `back-end` || `frontend` and `backend` || `front end` and `back end`

File: source/posts/2015-05-20-the-new-dockyard-dot-com.md (hunk 4 above)
Comment (rgbcolor) [inline source/posts/2015-05-20-the-new-dockyard-dot-com.md]: or maybe "for our clients' web applications." (changes meaning a bit)

File: source/posts/2015-05-20-the-new-dockyard-dot-com.md (hunk 3 above)
Comment (atreni) [inline source/posts/2015-05-20-the-new-dockyard-dot-com.md]: "but we consider ourselves to be first and foremost a User Experience company." (we are now considering seems to be future tense)

File: source/posts/2015-05-20-the-new-dockyard-dot-com.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+and [Phoenix](http://phoenixframework.org) on the backend.
```
Comment (rgbcolor) [inline source/posts/2015-05-20-the-new-dockyard-dot-com.md]: if front-end has a dash, should back-end have a dash too?

File: source/posts/2015-05-20-the-new-dockyard-dot-com.md (hunk 4 above)
Comment (kidfribble) [inline source/posts/2015-05-20-the-new-dockyard-dot-com.md]: "...delivering a modern user experience for web application to our clients." sounds very strange to me. I'm not quite sure how to change it but maybe as simple as:

"...delivering modern user experiences for our clients."

File: source/posts/2015-05-20-the-new-dockyard-dot-com.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+Over the next few weeks we're going to share the story on how our team rebuilt
+DockYard.com. You'll hear from our [Creative Director Steven Trevathan](http://twitter.com/strevat) on
+what went into the Discovery of our new website, the design decisions
+made, as well as a view of some early mockups and how we came to the
+final design. [Dan McClain](http://twitter.com/_danmcclain), our head of our Engineering team will discuss
+how we built the Ember application the "DockYard Way" which meant we
+actually had to make contributions back to Ember itself, as well as
+several other libraries, in order to get the experience just right.
+[Amanda Cheung](http://twitter.com/acacheung), our Lead UX Developer, will share what efforts went into the implementation of
+the design, how we build modern response markup. Finally you'll hear
```
Comment (poteto) [inline source/posts/2015-05-20-the-new-dockyard-dot-com.md]: design~~,~~ `and` how we build modern response markup

File: source/posts/2015-05-20-the-new-dockyard-dot-com.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+a company. We've always been considered an *engineering* company but we
+are now considering *ourselves* a **User Experience** company. All of
+our design and engineering decisions go into delivering a modern user
+experience for web application to our clients.
+
+The new website is also built how we believe modern web applications
+should be built: with [Ember.js](http://emberjs.com) on the front-end
+and [Phoenix](http://phoenixframework.org) on the backend.
+
+Over the next few weeks we're going to share the story on how our team rebuilt
```
Comment (rgbcolor) [inline source/posts/2015-05-20-the-new-dockyard-dot-com.md]: Maybe: the story **of** how

File: source/posts/2015-05-20-the-new-dockyard-dot-com.md (hunk 4 above)
Comment (kidfribble) [inline source/posts/2015-05-20-the-new-dockyard-dot-com.md]: or "...delivering modern web application user experiences for our clients"

File: source/posts/2015-05-20-the-new-dockyard-dot-com.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+DockYard.com. You'll hear from our [Creative Director Steven Trevathan](http://twitter.com/strevat) on
+what went into the Discovery of our new website, the design decisions
+made, as well as a view of some early mockups and how we came to the
+final design. [Dan McClain](http://twitter.com/_danmcclain), our head of our Engineering team will discuss
+how we built the Ember application the "DockYard Way" which meant we
+actually had to make contributions back to Ember itself, as well as
+several other libraries, in order to get the experience just right.
+[Amanda Cheung](http://twitter.com/acacheung), our Lead UX Developer, will share what efforts went into the implementation of
+the design, how we build modern response markup. Finally you'll hear
+from [Jon Lacks](http://twitter.com/jon_lacks), our Project Manager, will show how we managed the rebuild
```
Comment (poteto) [inline source/posts/2015-05-20-the-new-dockyard-dot-com.md]: Project Manager, `who` will

File: source/posts/2015-05-20-the-new-dockyard-dot-com.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+The new website is also built how we believe modern web applications
+should be built: with [Ember.js](http://emberjs.com) on the front-end
+and [Phoenix](http://phoenixframework.org) on the backend.
+
+Over the next few weeks we're going to share the story on how our team rebuilt
+DockYard.com. You'll hear from our [Creative Director Steven Trevathan](http://twitter.com/strevat) on
+what went into the Discovery of our new website, the design decisions
+made, as well as a view of some early mockups and how we came to the
+final design. [Dan McClain](http://twitter.com/_danmcclain), our head of our Engineering team will discuss
+how we built the Ember application the "DockYard Way" which meant we
```
Comment (atreni) [inline source/posts/2015-05-20-the-new-dockyard-dot-com.md]: suggestion: remove "which meant we" // replace with "actually making contributions back to Ember itself"

File: source/posts/2015-05-20-the-new-dockyard-dot-com.md [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+
+The new website is also built how we believe modern web applications
+should be built: with [Ember.js](http://emberjs.com) on the front-end
+and [Phoenix](http://phoenixframework.org) on the backend. 
+
+Over the next few weeks we're going to share the story on how our team rebuilt
+DockYard.com. You'll hear from our [Creative Director Steven Trevathan](http://twitter.com/strevat) on
+what went into the Discovery of our new website, the design decisions
+made, as well as a view of some early mockups and how we came to the
+final design. [Dan McClain](http://twitter.com/_danmcclain), our head of our Engineering team will discuss
```
Comment (rgbcolor) [inline source/posts/2015-05-20-the-new-dockyard-dot-com.md]: our head of Engineering 
or
the head of our Engineering

File: source/posts/2015-05-20-the-new-dockyard-dot-com.md [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+how we built the Ember application the "DockYard Way" which meant we
+actually had to make contributions back to Ember itself, as well as
+several other libraries, in order to get the experience just right.
+[Amanda Cheung](http://twitter.com/acacheung), our Lead UX Developer, will share what efforts went into the implementation of
+the design, how we build modern response markup. Finally you'll hear
+from [Jon Lacks](http://twitter.com/jon_lacks), our Project Manager, will show how we managed the rebuild
+of our own website using the exact same processes we use on our client
+projects.
+
+Along with these stories we'll not just tell you how we did things, we
```
Comment (poteto) [inline source/posts/2015-05-20-the-new-dockyard-dot-com.md]: we~~'ll~~ ~~not~~ `won't`

File: source/posts/2015-05-20-the-new-dockyard-dot-com.md [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+
+This new website is also built how we believe modern web applications
+should be built: with [Ember.js](http://emberjs.com) on the front-end
+and [Phoenix](http://phoenixframework.org) on the backend. 
+
+Over the next few weeks we're going to share the story of how our team rebuilt
+DockYard.com. You'll hear from our [Creative Director Steven Trevathan](http://twitter.com/strevat) on
+what went into the Discovery of our new website, the design decisions
+made, as well as a view of some early mockups and how we came to the
+final design. [Dan McClain](http://twitter.com/_danmcclain), our head of our Engineering team will discuss
```
Comment (rwjblue) [inline source/posts/2015-05-20-the-new-dockyard-dot-com.md]: > our head of our Engineering team will discuss

the head of our Engineering team

File: source/posts/2015-05-20-the-new-dockyard-dot-com.md (hunk 9 above)
Comment (atreni) [inline source/posts/2015-05-20-the-new-dockyard-dot-com.md]: who will show

File: source/posts/2015-05-20-the-new-dockyard-dot-com.md [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+
+With this relaunch of our website we are also changing our direction as
+a company. We've always been considered an *engineering* company but we
+are now considering ourselves a **User Experience** company. All of
+our design and engineering decisions go into delivering a modern user
+experience on the web for our clients.
+
+This new website is also built how we believe modern web applications
+should be built: with [Ember.js](http://emberjs.com) on the front-end
+and [Phoenix](http://phoenixframework.org) on the backend.
```
Comment (poteto) [inline source/posts/2015-05-20-the-new-dockyard-dot-com.md]: `back-end`

File: source/posts/2015-05-20-the-new-dockyard-dot-com.md (hunk 13 above)
Comment (rgbcolor) [inline source/posts/2015-05-20-the-new-dockyard-dot-com.md]: , the head of our engineering team, will discuss
(add comma, remove extra 'our')

File: source/posts/2015-05-20-the-new-dockyard-dot-com.md [hunk 15]
```diff
@@ -0,0 +1,48 @@
...
+are now considering ourselves a **User Experience** company. All of
+our design and engineering decisions go into delivering a modern user
+experience on the web for our clients.
+
+This new website is also built how we believe modern web applications
+should be built: with [Ember.js](http://emberjs.com) on the front-end
+and [Phoenix](http://phoenixframework.org) on the back-end. 
+
+Over the next few weeks we're going to share the story of how our team rebuilt
+DockYard.com. You'll hear from our [Creative Director Steven Trevathan](http://twitter.com/strevat) on
```
Comment (poteto) [inline source/posts/2015-05-20-the-new-dockyard-dot-com.md]: Creative Director [~~Creative Director~~ Steven Trevathan](http://twitter.com/strevat)

**Pull request merged**